src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
```

Comment:
Consider defining here helper functions, for example, isMan(), and helper constants, for example 
```const MAN = 'm'; ```
And use them further in your code instead of magic strings (considered programming antipattern) and code repeating.